core/src/test/java/com/sinch/sdk/core/databind/multipart/ObjectMapperTest.java

```diff
@@ -0,0 +1,260 @@
+            OptionalValue.of(fileAttachment1),
+            OptionalValue.of(Instant.parse("2024-11-25T09:06:54Z")),
+            OptionalValue.of(Arrays.asList("text 1", "text2")),
+            OptionalValue.of(Arrays.asList(AnEnum.from("a dyn value"), AnEnum.YES)),
+            OptionalValue.of(Arrays.asList(fileAttachment1, fileAttachment2)),
+            OptionalValue.of(
+                Arrays.asList(
+                    Instant.parse("2024-11-25T10:06:54Z"), Instant.parse("2024-11-25T09:06:54Z"))));
+
+    try {
+      serialized = new ObjectMapper().serialize(object);
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  @Test
+  void countValue() {
+    Assertions.assertEquals(8, serialized.size());
+  }
+
+  @Test
+  void order() {
+
+    String[] expectedOrder = {
+      SerializableObject.PROPERTY_RFC822_COLLECTION,
+      SerializableObject.PROPERTY_TEXT,
+      SerializableObject.PROPERTY_FILE_COLLECTION,
+      SerializableObject.PROPERTY_RFC822,
+      SerializableObject.PROPERTY_FILE,
+      SerializableObject.PROPERTY_ENUM,
+      SerializableObject.PROPERTY_ENUM_COLLECTION,
+      SerializableObject.PROPERTY_TEXT_COLLECTION
+    };
+    String[] keys = serialized.keySet().toArray(new String[0]);
+    TestHelpers.recursiveEquals(expectedOrder, keys);
+  }
+
+  @Test
+  void textValue() {
+    TestHelpers.recursiveEquals("text value", serialized.get("aText"));
+  }
+
+  @Test
+  void enumValue() {
+    TestHelpers.recursiveEquals("a dyn value", serialized.get("anEnum"));
+  }
+
+  @Test
+  void fileValue() {
+    TestHelpers.recursiveEquals(fileAttachment1, serialized.get("aFile"));
+  }
+
+  @Test
+  void rfc822Value() {
+    TestHelpers.recursiveEquals("Mon, 25 Nov 2024 09:06:54 GMT", serialized.get("aRfc822"));
+  }
+
+  @Test
+  void textCollectionValue() {
+    TestHelpers.recursiveEquals(
+        Arrays.asList("text 1", "text2"), serialized.get("aTextCollection"));
+  }
+
+  @Test
+  void enumCollectionValue() {
+    TestHelpers.recursiveEquals(
+        Arrays.asList(AnEnum.from("a dyn value"), AnEnum.YES), serialized.get("anEnumCollection"));
+  }
+
+  @Test
+  void fileCollectionValue() {
+    TestHelpers.recursiveEquals(
+        Arrays.asList(fileAttachment1, fileAttachment2), serialized.get("aFileCollection"));
+  }
+
+  @Test
+  void rfc822CollectionValue() {
+    TestHelpers.recursiveEquals(
+        new ArrayList<>(
+            Arrays.asList("Mon, 25 Nov 2024 10:06:54 GMT", "Mon, 25 Nov 2024 09:06:54 GMT")),
+        serialized.get("aRfc822Collection"));
+  }
+
+  @PropertiesOrder({
+    SerializableObject.PROPERTY_RFC822_COLLECTION,
+    SerializableObject.PROPERTY_TEXT,
+    SerializableObject.PROPERTY_FILE_COLLECTION,
+    SerializableObject.PROPERTY_RFC822,
+    SerializableObject.PROPERTY_FILE,
+    SerializableObject.PROPERTY_ENUM,
+    SerializableObject.PROPERTY_ENUM_COLLECTION,
+    SerializableObject.PROPERTY_RFC822_COLLECTION,
```

Comment:
Is it expected to have this property duplicated?

Reply:
Absolutely not.
Fixed within this PR/commit: https://github.com/sinch/sinch-sdk-java/pull/171/commits/d45ce941c4e56f52515c035439f129c7236277ce